services/galley/src/Galley/API/Teams.hs

```diff
@@ -190,7 +197,11 @@ uncheckedAddTeamMember :: TeamId ::: Request ::: JSON ::: JSON -> Galley Respons
 uncheckedAddTeamMember (tid ::: req ::: _) = do
     nmem <- fromBody req invalidPayload
     mems <- Data.teamMembers tid
-    addTeamMemberInternal tid Nothing Nothing nmem mems
+    rsp <- addTeamMemberInternal tid Nothing Nothing nmem mems
+    let membs = nmem^.ntmNewTeamMember : mems
+    let bUsers = view userId <$> filter (`hasPermission` SetBilling) membs
+    journal $ Journal.teamUpdate tid (fromIntegral $ length membs) bUsers
```

Comment:
I noticed that 
```
let membs = filter (\u -> u^.userId /= remove) mems
let bUsers = view userId <$> filter (`hasPermission` SetBilling) membs
 $ Journal.teamUpdate tid (fromIntegral $ length membs) bUsers
```
is a recurring pattern, with only slight variation on how `membs` is constructed. Can we refactor that?